src/app/multisender/types.ts

```diff
@@ -0,0 +1,12 @@
+export const supportedChains = ["Ethereum", "ETH Sepolia"] as const;
+export const supportedCurrencies = ["ETH", "ETH Sepolia", "USDC", "USDT"] as const;
+
+export type SupportedChain = (typeof supportedChains)[number];
+export type supportedCurrency = (typeof supportedCurrencies)[number];
+
+export type TokenTransferInfo = {
```

Comment:
Remove chain and currency from the csv and list. This will be a selector either before or after adding addresses